curvefs/src/client/inode_wrapper.cpp

```diff
@@ -465,7 +467,7 @@ void InodeWrapper::AsyncS3(MetaServerClientDone *done, bool internal) {
         metaClient_->UpdateInodeWithOutNlinkAsync(
             inode_, done, InodeOpenStatusChange::NOCHANGE, &s3ChunkInfoAdd_);
         dirty_ = false;
-        s3ChunkInfoAdd_.clear();
+        ClearS3ChunkInfoAdd();
```

Comment:
there's a trap when updating it asynchronously, we copy `s3ChunkInfoAdd_` into the async task, so it's still cost memory and it's been released after the task succeeded

Reply:
Yes, here the s3ChunkInfoMetric mainly records the s3chunkinfo size in iCache and which in inflight request is temporary and will released when request finished.